docs/tutorial/tutorial.md

```diff
@@ -0,0 +1,129 @@
+# Tutorial
+
+### Prerequisites
+
+To deploy wordpress-k8s charm, you will need a juju bootstrapped with any kubernetes controller.
+To see how to bootstrap your juju installation with microk8s, please refer to the documentation
+on microk8s [installation](https://juju.is/docs/olm/microk8s).
+
+### Setting up the environment
+
+To easily clean up the resources and to separate your workload from the contents of this tutorial,
+it is recommended to set up another environment by adding a new model with the following command.
+
+```
+juju add-model wordpress-tutorial
+```
+
+### Deploy wordpress-k8s charm
+
+Deployment of WordPress requires a database integration. A preferred method in juju is through the
+native juju relation. In this case, a `mysql` interface is required by the wordpress-k8s charm and
+hence, [`mysql-k8s`](https://charmhub.io/mysql-k8s) charm will be used.
+
+Start off by deploying the wordpress charm. By default it will deploy the latest stable release of
+the wordpress-k8s charm.
+
+```
+juju deploy wordpress-k8s
+```
+
+If you wish to try out the latest features of the wordpress-k8s charm you can set an additional
+parameter to deploy from a specific channel. For example, to deploy from the edge channel:
+
+```
+juju deploy wordpress-k8s --channel=edge
+```
+
+### Deploy and relate database
+
+#### Database relation
+
+The following commands deploys the mysql-k8s charm and relates wordpress-k8s charm through its mysql
+interface.
+
+```
+juju deploy mysql-k8s \
+    --config mysql-interface-user=wordpress \
+    --config mysql-interface-database=test-database
+juju relate wordpress-k8s mysql-k8s
+```
+
+#### Database configuration
+
+Note: If you’ve completed the steps in the Database relation section, you may skip this part. If
```

Comment:
Would it be helpful to be clear that this is only required if an integration is not used? `may skip` sounds like it could still be useful to do this part of the tutorial

Reply:
Clarified the sentence
`This section is required only if database relation is not used. `
Thank you for the suggestion!